packages/smart-vaults/src/utils/LightSyncMultiSigner.sol

```diff
@@ -84,56 +57,53 @@ abstract contract LightSyncMultiSigner is MultiSigner {
     /// @dev reverts if validation fails.
     function _validateSignerSetUpdate(
         MultiSignerLib.MultiSignerStorage storage $_,
-        SignerSetUpdate memory signerUpdate_,
+        SignerSetUpdate memory signerSetUpdate_,
         uint256 nonce_
     )
         internal
         view
     {
         if (
             !MultiSignerSignatureLib.isValidSignature(
-                $_, _getSignerUpdateHash(signerUpdate_, nonce_), signerUpdate_.normalSignature
+                $_, _getSignerUpdateHash(signerSetUpdate_, nonce_), signerSetUpdate_.normalSignature
             )
-        ) revert SignerSetUpdateValidationFailed(signerUpdate_);
+        ) revert SignerSetUpdateValidationFailed(signerSetUpdate_);
```

Comment:
does this cause an issue for dummy sig gas estimation?

Reply:
no, we would already have the right signature for this.